src/pages/Details.vue

```diff
@@ -76,7 +77,7 @@
                 <div class="row">
                     <div class="col-md-8">
                         <HeartbeatBar :monitor-id="monitor.id" />
-                        <span class="word">{{ $t("checkEverySecond", [ monitor.interval ]) }}</span>
+                        <div><span v-if="monitor.active" class="mr-2">Next in {{ formattedTime }}&nbsp;</span><span class="word">({{ $t("checkEverySecond", [ monitor.interval ]) }})</span></div>
```

Comment:
I am not shure, but I would put the "Next in .." inside the brackets. How do other notification systems handling this?

An alternative might be to underline the seconds which this is checked by and display a popover to help if it is hovered over.

What do you think?

Reply:
done made the `next in` be inside the brackets, popover might be an overkill for this as we have enough space which is laying empty, and when implementing popever we would also have to think about mobile users